src/main/java/gov/loc/repository/bagit/creator/AbstractCreateManifestsVistor.java

```diff
@@ -50,14 +64,37 @@ public FileVisitResult visitFile(final Path path, final BasicFileAttributes attr
       logger.debug("Skipping [{}] since we are ignoring hidden files", path);
     }
     else{
-      Hasher.hash(path, manifestToMessageDigestMap);
+      try(final InputStream is = new BufferedInputStream(Files.newInputStream(path, StandardOpenOption.READ))){
+        final byte[] buffer = new byte[CHUNK_SIZE];
+        int read = is.read(buffer);
```

Comment:
Since read is a pretty common term - I count three different uses in as many lines here - maybe this should be something like `bytesRead` for easier reading?